vallabhpadhye.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "vallabhpadhye",
+	"twitter": "vallabhpadhye"
+  },
+  "records": {
+    "CNAME": " vallabhpadhye.github.io"

```

it seems you are redirecting it in your gh profile, we also support redirection:
```suggestion
    "URL": "https://github.com/VallabhPadhye"
```